### DISCUSSION TITLE
Help opening 5250 terminal

### DISCUSSION THREAD

**Discussion by felipunkz** [codefori/vscode-ibmi#2336] [Q&A] [answered]
Hi guys.

I hope you can help me with an issue I'm having.

I'm using the Code for IBM i extension, and I want to open a 5250 terminal, I've followed the instructions from the documentation, and I have installed the tn5250 package on the IBM i system. I've also added the path to my .profile file and can confirm that if I connect using ssh without doing anything else I can invoke the tn5250 command, but in VS Code I'm still receiving the message "5250 terminal is not supported. Please install tn5250 via yum on the remote system." do you know what else am I missing?

I've already reconnect to my system after installing tn5250 and setting the path variable, but I still have no luck opening the terminal, PASE terminal works as expected.

Thanks in advance.

**Comment by sebjulliand** [comment, marked as answer]
Hi!
Did you try reconnecting and reloading the server settings? If not, that's certainly what's missing before this can work for you 😊
![image](https://github.com/user-attachments/assets/1aab1bed-13d8-44f6-aca1-c73afd6c3af8)

**Reply by felipunkz** [reply]
Thanks @sebjulliand !!! I've kept researching the option and that is actually what I've found, I was about to answer that, and Booom! there's your response, that made the trick! thank you very much for your attention and help!

**Reply by sebjulliand** [reply]
You're very welcome 😉
